### PR TITLE
Fix fetching community members revealed addresses

### LINKED ISSUE #11143: Display community members in token holders list and implement actions
The mapping:
wallet <-> community members mapping
is probably already implemented (community.Members()[pubkey].RevealedAccounts).
We need to use it in order to show which holders are community members and also to implement some actions from issue: https://github.com/status-im/status-desktop/issues/11033

Function for getting balances for a list of wallet accounts: GetTokensBalancesForChainIDs
Need to fill member name and image in initTokenOwnersItem().

### PR DESCRIPTION
**Pull request by MishkaRogachev**
Important changes:
- [x] Fix fetching revealed accounts in `GetCommunityMembersForWalletAddresses`

Closes https://github.com/status-im/status-desktop/issues/11143

### CODE REVIEW

File: protocol/messenger_communities.go [hunk 1]
```diff
@@ -4380,14 +4380,19 @@ func (m *Messenger) GetCommunityMembersForWalletAddresses(communityID types.HexB
...
-			if slices.Contains(revealedAddress.ChainIds, chainID) {
-				contact, ok := m.allContacts.Load(memberID)
+	for _, memberPubKey := range community.GetMemberPubkeys() {
+		memberPubKeyStr := common.PubkeyToHex(memberPubKey)
+		revealedAccounts, err := m.communitiesManager.GetRevealedAddresses(communityID, memberPubKeyStr)
+		if err != nil {
+			return nil, err
+		}
+		for _, revealedAccount := range revealedAccounts {
+			if slices.Contains(revealedAccount.ChainIds, chainID) {
```
Comment (igor-sirotin) [inline protocol/messenger_communities.go]: I'd appreciate short-circuit continue, but that's not important. Just less spaghetti 🙂 
```go
if !slices.Contains(revealedAccount.ChainIds, chainID) {
    continue
}
...
```

**Review by igor-sirotin — APPROVED** [review]
I didn't get what was the problem with old code, but the LGTM as well 🙂

**Review by jrainville — APPROVED** [review]

**Comment by jrainville** [comment]
> I didn't get what was the problem with old code, but the LGTM as well slightly_smiling_face

@igor-sirotin the revealed addresses are no longer part of the community description. We need to get it from another BD table now

**Review by osmaczko — APPROVED** [review]

File: protocol/messenger_communities.go (hunk 1 above)
Comment (MishkaRogachev) [reply protocol/messenger_communities.go]: fixed!

**Pull request merged**